vnext/Microsoft.ReactNative/Base/CoreUIManagers.cpp

```diff
@@ -29,9 +29,6 @@
 
 // Polyester View Managers // TODO: Move Polyester implementations out of this
 // library and depot
```

Comment:
maybe you can delete this comment now :)

consider moving the polyester IconViewManager outside of the polyester folder and maybe renaming it to GlyphViewManager since that's what we call it publicly